pkg/ansible/watches/watches.go

```diff
@@ -72,7 +69,7 @@ type Finalizer struct {
 var (
 	blacklistDefault                   = []schema.GroupVersionKind{}
 	maxRunnerArtifactsDefault          = 20
-	reconcilePeriodDefault             = "0s"
+	reconcilePeriodDefault             = metav1.Duration{Duration: time.Duration(0)}
```

Comment:
```suggestion
	reconcilePeriodDefault             = time.Second * 0
```

Reply:
This throws an error. I tried to do this and many other workarounds. This is the only thing that works for me.

Reply:
It has to be of the ```metav1``` format